src/screens/OtherProfile/elements/BioAndChat.js

```diff
@@ -145,24 +145,24 @@ const BioAndChat = (props) => {
 
 const styles = StyleSheet.create({
   bioAndSendChatContainer: (isAnonimity) => ({
-    backgroundColor: isAnonimity ? colors.bondi_blue : colors.blue1,
+    backgroundColor: isAnonimity ? COLORS.anon_primary : COLORS.signed_primary,
     borderRadius: 15,
     paddingHorizontal: 10,
     paddingTop: 10
   }),
   containerBio: {
     marginBottom: 10
   },
-  bioText: (isAnonimity) => ({
-    color: isAnonimity ? colors.greenDark : colors.white,
+  bioText: {
+    color: COLORS.white,
     fontSize: 14,
     fontWeight: '600',
     lineHeight: 22
-  }),
+  },
   seeMore: {
     fontFamily: fonts.inter[500],
     fontSize: 14,
-    color: colors.black
+    color: COLORS.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles have been updated to use the new `COLORS` object. The `bioAndSendChatContainer` and `bioText` styles no longer take `isAnonimity` as a parameter, which could potentially change the appearance of these elements. Please ensure that this change is intentional and does not negatively affect the UI.
```diff
- backgroundColor: isAnonimity ? colors.bondi_blue : colors.blue1,
+ backgroundColor: isAnonimity ? COLORS.anon_primary : COLORS.signed_primary,

- bioText: (isAnonimity) => ({
- color: isAnonimity ? colors.greenDark : colors.white,
+ bioText: {
+ color: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->